master/internal/api_user.go

```diff
@@ -683,3 +696,253 @@ func (a *apiServer) PostUserActivity(
 	}
 	return &apiv1.PostUserActivityResponse{}, err
 }
+
+// POST Long lived token takes optional lifespan and overwrites an already existing long
+// lived token for the current logged in user.
+func (a *apiServer) PostLongLivedToken(
```

Comment:
I am yet to change from `PostLongLivedToken` to `PostLongLivedAccessToken`.
Will updated it based on the decisions made during the call with the product.

Reply:
Ideally we are looking forward to have them as `PostAccessToken`, will be changing them in next iteration.